Assets/scss/components/quesionnaire.scss

```diff
@@ -0,0 +1,276 @@
+.cmp-questionnaire {
+  max-width: toRem(1175);
+  margin: auto;
+
+  @include max-screen(sm) {
+    max-width: toRem(767);
+    margin: auto;
+  }
+
+  @include breakpoint-between(sm, md) {
+    max-width: toRem(1199);
+    margin: auto;
+  }
+
+  &__container {
+    display: flex;
+    background-color: $grey94;
+    height: toRem(590);
+
+    @include max-screen(sm) {
+      display: flex;
+      flex-direction: column;
+      height: 164vh;
+      overflow: auto;
+      background-color: $grey94;
+    }
+
+    @include breakpoint-between(sm, md) {
+      height: 100vh;
+      overflow: auto;
+      display: flex;
+      flex-direction: column;
+    }
+  }
+
+  &__image {
+    max-width: 50%;
+    height: toRem(590);
+
+    @include max-screen(sm) {
+      max-width: 100%;
+      object-fit: cover;
+    }
+
+    @include breakpoint-between(sm, md) {
+      max-width: 100%;
+      object-fit: cover;
+    }
+  }
+
+  &__form {
+    margin-block-start: toRem(40);
+    margin-inline: toRem(40) toRem(50);
+
+    @include breakpoint-between(sm, md) {
+      margin-inline-end: toRem(100);
+    }
+  }
+
+  &__title {
+    text-align: left;
+    margin-block-end: toRem(30);
+
+    @include breakpoint-between(sm, md) {
+      text-align: center;
+    }
+  }
+
+  &__heading {
+    line-height: 1.32;
+    text-wrap: balance;
+    font-size: toRem(24);
+    font-weight: $font-bold;
+    font-style: normal;
+    color: $black;
+
+    @include max-screen(sm) {
+      font-size: toRem(33);
+      font-weight: 400;
+    }
+
+    @include breakpoint-between(sm, md) {
+      text-align: center;
+      font-size: toRem(60);
+      font-weight: 500;
+    }
+  }
+
+  &__content {
+    margin-block-start: toRem(8);
+    line-height: 1.4;
+    text-wrap: balance;
+    font-size: toRem(16);
+    color: $black;
+
+    @include breakpoint-between(sm, md) {
+      text-align: center;
+      font-size: toRem(24);
+      font-weight: 400;
+    }
+  }
+
+  &__fields {
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+
+    @include breakpoint-between(sm, md) {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+    }
+  }
+
+  &__inputfields {
+    display: grid;
+    grid-template-columns: toRem(250) toRem(250);
```

Comment:
Since this is the same value for both columns, you could use [repeat](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat):

```css
grid-template-columns: repeat(2, toRem(250));
```

Reply:
Thanks for the Feedback this has been addressed